app/views/admins/dashboard.mobile.haml

```diff
@@ -0,0 +1,4 @@
+- content_for :head do
+  = stylesheet_link_tag :admin
+
+= render partial: "admins/dashboard"
```

Comment:
In mobile Dashboard now I have a message " Determining latest diaspora* version... " that is never actually get replaced with anything else. In desktop version it determines the newest version and changes its background to green. If the version check doesn't work for mobile these "Determining latest diaspora* version... " words perhaps shouldn't be rendered or it should say "Cannot determine latest diaspora* version" and maybe just display the current version.